publish.sbt

```diff
@@ -0,0 +1,48 @@
+import ReleaseTransformations._
+
+// Your profile name of the sonatype account. The default is the same with the organization value
+sonatypeProfileName := "uk.sky"
+
+// To sync with Maven central, you need to supply the following information:
+publishMavenStyle := true
+
+// Open-source license of your choice
+licenses += ("BSD New", url("https://opensource.org/licenses/BSD-3-Clause"))
+
+homepage := Some(url("https://github.com/sky-uk/kafka-topic-loader"))
+scmInfo := Some(
+  ScmInfo(
+    url("https://github.com/sky-uk/kafka-topic-loader"),
+    "scm:git@github.com:sky-uk/kafka-topic-loader.git"
+  )
+)
+developers := List(
+  Developer(id = "bcarter97",
+            name = "Ben Carter",
+            email = "ben.carter@sky.uk",
+            url = url("https://github.com/bcarter97"))
+)
```

Comment:
Is this required for maven central? Whats its purpose?

Reply:
As for the purpose, it's to list the devs in the sonatype repo, e.g.: https://search.maven.org/artifact/uk.sky/cqlmigrate/0.12.0/jar

Reply:
Should probably look at what other companies do. I don't expect each single dev to be listed there unless it's an individual contributor.

Reply:
apache use the GithubHosting option for one of their projects:

https://github.com/apache/incubator-marvin/blob/2ac8a7f37af8626d47a6896b1d3e6646793f41e9/engine-executor/sonatype.sbt#L7

This lists the "developer" as the application name, which looks like [this](https://search.maven.org/artifact/org.apache.marvin/marvin-engine-executor_2.12/0.0.5/jar) in the maven repo. Plugin requires the email, so either we can use a designated email or put a dummy one in as they have

Reply:
[Other examples](https://github.com/search?q=Some%28GitHubHosting%28&type=code) either use a support email or an individual dev's email

Reply:
`GithubHosting` looks promising, given that it would allow us not to have to maintain a developer list. How `marvin-engine-executor` is configured looks wrong though.